docs/_templates/compiler_version.tmpl

```diff
@@ -0,0 +1,24 @@
+.. -*- mode: rst -*-
+
+{% if data['compiler_version']%}
+
+Compiler version
+~~~~~~~~~~~~~~~~
+
+Specifying the compatible compiler version for a V1 module is done via the ``compiler_version`` field of the ``module.yml``
+file.
```

Comment:
I just noticed that this field is nowhere explained in the documentation. Do you create a ticket to explain what the compiler_version field is about.

Reply:
Decided to drop the compiler section as per https://inmanta.slack.com/archives/CKRF0C8R3/p1713441521052269?thread_ts=1713339151.435429&cid=CKRF0C8R3